{"uuid": "ec2417ed-5d77-456b-929f-c55cc05ca8b4", "children": ["275165b3-5ee6-4ff2-b40b-62571f3b5b42"], "befores": [{"name": "_xunit_setup_class_fixture_Test_plugins", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable\n", "trace": "  File \"D:\\python 3.8.7\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\python.py\", line 864, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\python.py\", line 785, in _call_with_optional_argument\n    func(arg)\n  File \"d:\\wanwan1\\test_plugins.py\", line 19, in setup_class\n    self.p1.initialize()\n  File \"d:\\wanwan1\\login.py\", line 146, in initialize\n    self.send_cmd(\"tftp -g -r testserver %s\"(self.pcip))\n"}, "start": 1700738484612, "stop": 1700738506913}], "start": 1700738484612, "stop": 1700738507051}